src/Illuminate/Support/Arr.php

```diff
@@ -120,7 +120,7 @@ public static function exists($array, $key)
             return $array->offsetExists($key);
         }
 
-        return array_key_exists($key, $array);
+        return isset($array[$key]) || array_key_exists($key, $array);
```

Comment:
There's a benchmark example in [php.net](http://php.net/manual/en/function.array-key-exists.php)

> Benchmark (100000 runs):
array_key_exists() : 205 ms
isset() : 35ms
isset() || array_key_exists() : 48ms

However if you are running this 100000 times i guess that 150ms is not your main concern.